### ISSUE TITLE
linkedin: profile not found when using cookies

### ISSUE THREAD

**Issue by wassim-azirar** [linkedtales/scrapedin#54]
When I use the scrapper with login/password options it works fine.
But when I use my cookie, I get an error of type 'linkedin: profile not found'
```javascript
const fs = require('fs');
const scrapedin = require("scrapedin");

// const options = { email: "name@gmail.com", password: "abcdefgh" };
const cookies = fs.readFileSync('./cookie.txt');
const options = {
  cookies: JSON.parse(cookies)
}

console.log(options);

const url = 'https://www.linkedin.com/in/wassimazirar/';
scrapedin(options)
    .then(profileScraper => profileScraper(url, 2000))
    .then(profile => {
        console.log(profile);
    });
```

```batch
(node:13224) UnhandledPromiseRejectionWarning: Error: linkedin: profile not found
    at ..\node_modules\scrapedin\src\profile.js:19:13
    at async module.exports (..\node_modules\scrapedin\src\profile.js:16:3)
(node:13224) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:13224) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by wassim-azirar** [comment]
@leonardiwagner did you check my issue ?
Thanks in advance :confused:

**Comment by leonardiwagner** [comment]
Hi @wassim-azirar , sorry for the late response:
- Please turn on `isHeadless` option ([see documentation](https://github.com/linkedtales/scrapedin/wiki/Documentation#--options-object)) and check in the automated browser if you are really logged when it's scrapping
- Increase `waitTimeMs` ([see documentation](https://github.com/linkedtales/scrapedin/wiki/Documentation#--profilescraperurl-waittimems--500-function)) to ensure that LinkedIn already loaded the page before starting scraping. I personally use `5000`

Since you say that via email/password is working I suspect that's something wrong with your cookies approach.

I also suspect there's a "warmup time" from LinkedIn when using cookies for first time, so both increasing the wait time and you checking what's happening in the browser will solve your problem, please give me feedback after trying these steps, thank you.

**Comment by wassim-azi** [comment]
Hello @leonardiwagner 

I changed my code to wait for 5000 ms

```javascript
const cookies = fs.readFileSync('./cookie.txt');
const options = {
  cookies: JSON.parse(cookies),
  isHeadless: true,
  hasToLog: true
};

scrapedin(options)
    .then(profileScraper => profileScraper(url, 5000))
    .then(profile => { 
        response.end(JSON.stringify(profile)); 
    });
```
I also turned on debugging and logged in / logged out several times and exported my cookies using multiple chrome extensions. But I always get the same result.

```batch
scrapedin: 2019-11-13T20:21:36.646Z info: [scrapedin] initializing
scrapedin: 2019-11-13T20:21:36.982Z info: [scrapedin] using cookies, login will be bypassed
scrapedin: 2019-11-13T20:21:36.989Z info: [profile] starting scraping url: https://www.linkedin.com/in/username
scrapedin: 2019-11-13T20:21:43.963Z warn: [profile] profile selector was not found
(node:9952) UnhandledPromiseRejectionWarning: Error: linkedin: profile not found
    at .\node_modules\scrapedin\src\profile.js:19:13
```

I can give you my login/password or my exported cookies in private if you want to test.

**Comment by jessica-yang** [comment]
Just wanted to comment that I'm facing the same issue.

**Comment by wassim-azirar** [comment]
Any news about this issue ?

**Comment by leonardiwagner** [comment]
@wassim-azirar @jessica-yang new version released, please update to `v1.0.11`.

Possible fixes for this issue:
- We had a bug: [setCookies method wasn't "awaited"](https://github.com/linkedtales/scrapedin/commit/58c2954cca6a660796d7f90b392d036b7e364f98#diff-263d236a9b2f1f1085bdf7244f8da326R17) 
- Not finding profile selector [doesn't throw an error anymore](https://github.com/linkedtales/scrapedin/commit/58c2954cca6a660796d7f90b392d036b7e364f98#diff-cbe8337fe566c79fb8fda239fdae2b73L20), it'll be a warning.


please update the version and give a feedback about the issue, thanks

**Comment by jessica-yang** [comment]
Thanks @leonardiwagner ! Looks like it's working now.
